source/manual/create-a-local-transaction.html.md

```diff
@@ -0,0 +1,236 @@
+
+```bash
+bundle exec rails import:service_links[LGSL_CODE,LGIL_CODE,REMOTE_FILE_PATH]
+```
+
+## Add content and publish the service in Publisher
+
+Once __all__ of the above steps are complete, ask a member of your content design team to add any desired content and publish the service in [Publisher](https://publisher.publishing.service.gov.uk). The content design team will need to know the [LGSL code](#lgsl-code), the [LGIL code](#lgil-code), and that the page needs to be created as a new `Local transaction` format in publisher.
+
+## Glossary
+
+### LGSL code
+
+A Local Government Services List (LGSL) is a number that LLM uses to uniquely identify a service. They originate in ESD where it is used as the service ID.
+
+### LGIL code
+
+Along with services, ESD has the concept of an Interaction. Functionally, the relationship between service and interaction is many-to-many.
+
+An interaction describes the reason, use, or purpose of the service.
+
+Like an [LGSL code](#lgsl-code), LLM uses the interaction ID from ESD to uniquely identify an interaction.  LLM this the Local Government Interaction List (LGIL) code.
+
+One thing to note is that ESD has deprecated the use of interactions. LLM has not however been updated to accommodate the replacement concept in ESD. This means that services in LLM still need a relationship with one-or-more interaction in order to function correctly. If you are unsure which interaction(s) it is that you need, it is recommended that the Providing information (LGIL code 8) interaction be used as this is the generic in nature.
+
+### Seek content design advice
+
+For service details such as: `name`, `description` and `slug` - you must get a content designer to create them. It is highly unlikely that the details from ESD will work and the `slug` will be used for the URL of the service on GOV.UK.
+
+For unavailability elements such as: `title`, `body`, `button_text` and `button_link` - there is standard text available [here](https://docs.google.com/document/d/1V1oP9O25IAudOsM168sYbsOegZfRZgPAJvdVrRxoEyY/edit#).
+
+### Providing tier
+
+A providing tier is way of identifying the [levels](https://www.gov.uk/understand-how-your-council-works) at which a service is implemented by a local authority. A tier can be either:
+
+* `county/unitary`
+* `district/unitary`
+* `all`
+
+In LLM each tier has a nominal level indicator that makes them hierarchical in nature.
+
+1. County
+2. District
+3. Unitary
+
+This is important as, in order to present the most appropriate URL for a service to the user, the tiers are checked for a match starting at the lowest level (3) working up to the highest level (1). This way http://www.coventry.gov.uk/a-specific-serivce would be presented before http://www.coventry.gov.uk for example.
+
+If the providing tier for a service is unknown, using `all` is recommended as this offers the best chance of finding the most appropriate URL.
+
+## Create a new service
```

Comment:
This content is only referred to in one place, so perhaps it should be moved into that section to avoid having to jump around the guidance?

Reply:
Have moved this up into the section that referenced it.  Thanks.